docs/source/traitsui_user_manual/factories_basic.rst

```diff
@@ -1177,6 +1267,23 @@ You can specify a mapping from user input values to other values with the
 by passing a reference to it in the *evaluate* parameter, or by passing the
 extended name of a trait that references it in the *evaluate_name* parameter.
 
+
+TimeEditor()
+````````````
+
+:Suitable for:
+    Time
+:Default for:
+    Time
+:Optional parameters:
+    *minimum_datetime*, *strftime*
+
+The TimeEditor() displays a Python datetime time object, usually supplied
```

Comment:
Same comment as for "datetime date"; I think this is potentially confusing unless it's clear that "datetime" refers to the datetime module.